tutorials/explainers/KernelSHAP/kernelshap_geometric_shapes.ipynb

```diff
@@ -49,7 +49,7 @@
   },
```

Comment:
<p>Nice. Work. Still, it would be best to first show the "true" class, that is the "triangle" like it was before IMHO.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/805/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Fixed!